packages/fetch-and-compile/lib/types.ts

```diff
@@ -1,5 +1,16 @@
 import type { WorkflowCompileResult } from "@truffle/compile-common";
-import type { SourceInfo } from "@truffle/source-fetcher";
+import type {
+  SourceInfo,
+  NetworkInfo as FetcherNetworkInfo
+} from "@truffle/source-fetcher";
+
+export interface NetworkInfo extends FetcherNetworkInfo {
+  fetchers: string[];
+}
+
+export interface NetworkInfos {
```

Comment:
Why don't we just call this `SupportedNetworks`? Since it's the return type of `getSupportedNetworks()`?

Reply:
Sure, that would make sense.